Bottom: eb2ed0554ae828c1065eafc7f847e77933cf4ce8
Top:    fbf620e3ecc9bc797c0a25b967186f3b3adcec5d
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-28 21:30:38 +0100

Refresh of determine-layer-count-in

---

diff --git a/src/parse_y.y b/src/parse_y.y
index 3607677..9ab75ce 100644
--- a/src/parse_y.y
+++ b/src/parse_y.y
@@ -210,26 +210,19 @@ parsepcb
 			PCB = pcb_save;
 			}
 			   
-<<<<<<< current
 		| {
 		    yyFont = &yyPCB->Font;
 		    yyData = yyPCB->Data;
 		    yyData->pcb = yyPCB;
 		    yyData->LayerN = 0;
-		    layer_group_string = NULL;
-		}
-=======
-		| { PreLoadElementPCB (); }
->>>>>>> patched
+		  }
 		  element
 		  {
 		    PCBType *pcb_save = PCB;
 		    ElementType *e;
 
-		    yyData->LayerN = 2;
-
 		    CreateNewPCBPost (yyPCB, 0);
-		    ParseGroupString("1,c:2,s", &yyPCB->LayerGroups, yyData->LayerN);
+		    ParseGroupString("1,c:2,s", &yyPCB->LayerGroups, &yyData->LayerN);
 		    e = yyPCB->Data->Element->data; /* we know there's only one */
 		    PCB = yyPCB;
 		    MoveElementLowLevel (yyPCB->Data, e, -e->BoundingBox.X1, -e->BoundingBox.Y1);
